compiler/src/iree/compiler/Codegen/LLVMGPU/LLVMGPUVectorLowering.cpp

```diff
@@ -183,6 +183,261 @@ struct SetMulAddFMF final : OpRewritePattern<vector::MultiDimReductionOp> {
   }
 };
 
+struct ContractToChainFMA final : OpRewritePattern<vector::ContractionOp> {
+  using OpRewritePattern::OpRewritePattern;
+
+  LogicalResult matchAndRewrite(vector::ContractionOp op,
+                                PatternRewriter &rewriter) const override {
+    // TODO: add rewrite to support relevant contractions nested in vector.mask
```

Comment:
nit: follow the llvm coding standards on comments: https://llvm.org/docs/CodingStandards.html#commenting
```suggestion
    // TODO: Add a rewrite to support relevant contractions nested in `vector.mask`.
```